### ISSUE TITLE
[HOLD for payment 2025-01-28] [Hold 54760] [HOLD for payment 2025-01-15] [$250] [Per Diem] Frontend - Creating Per Diem expenses in Newdot[

### ISSUE THREAD

**Issue by Gonals** [Expensify/App#50868]
# Part of the _Add Per Diem to New Expensify_ project
Main issue: https://github.com/Expensify/Expensify/issues/400276
Doc section: https://docs.google.com/document/d/1c6WDOJlhvODD0mRKaCHfQk_HRNmq5K7hSDHLu06Pbq8/edit?tab=t.0#bookmark=id.aozt511io0w8
Project: https://github.com/Expensify/Expensify/issues/400276

# Feature Description
Add the necessary changes to allow Per Diem expense creation in newdot

# Manual Test Steps

# Automated Tests
<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~021846477939875217088</li>
        <li>Upwork Job ID: 1846477939875217088</li>
        <li>Last Price Increase: 2024-10-16</li>
    </ul>
</details>

<details><summary>Issue Owner</summary>Current Issue Owner: @</details>

<details><summary>Issue Owner</summary>Current Issue Owner: @Christinadobrzyn</details>

**Comment by Gonals** [comment]
Not overdue

**Comment by c3024** [comment]
I am due for payment as C+ for reviews of PRs #54760 and #54252 here.

**Comment by Christinadobrzyn** [comment]
Sounds good - @shubham1206agra and @c3024 can you make sure this [payment summary](https://github.com/Expensify/App/issues/50868#issuecomment-2603924150) is correct? 

Do we need a regression test for this?

**Comment by c3024** [comment]
@Christinadobrzyn 

No, this is a part PR for a feature in works. This issue does not need a regression test.

**Comment by JmillsExpensify** [comment]
$250 approved for @c3024

**Comment by Christinadobrzyn** [comment]
Perfect! Thank you @c3024 and @shubham1206agra! Closing this as complete.
